lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -77,6 +80,7 @@ export default class BitcoinProcessor {
 
   public constructor (config: IBitcoinConfig) {
     this.bitcoinPeerUri = config.bitcoinPeerUri;
+    this.bitcoinAuthorization = Buffer.from(`${config.bitcoinRpcUsername}:${config.bitcoinRpcPassword}`).toString('base64');
```

Comment:
It would be nice if the authentication is optional instead.

Reply:
While I have made it optional, the documentation will treat RPC authorization as required as the satoshi client does [require an rpc password](https://bitcoin.org/en/developer-reference#remote-procedure-calls-rpcs).